okapi-core/src/test/java/org/folio/okapi/service/impl/DockerModuleHandleTest.java

```diff
@@ -87,28 +80,27 @@ public void testDockerVersionAtLocal(TestContext context) {
     DockerModuleHandle dh = new DockerModuleHandle(vertx, ld,
       "mod-users-5.0.0-SNAPSHOT", ports, "localhost", 9232, new JsonObject());
 
+    JsonObject versionRes = new JsonObject();
+    Async async = context.async();
     dh.getUrl("/version", res -> {
-      Assume.assumeTrue(res.succeeded());
-      if (res.failed()) {
-        logger.warn(res.cause().getMessage());
-        async.complete();
-        return;
+      if (res.succeeded()) {
+        versionRes.put("result", res.result());
       }
-      dh.deleteUrl("/version", "msg", res2 -> { // provoke 404 not found
-        context.assertTrue(res2.failed());
-        if (res2.failed()) {
-          context.assertTrue(res2.cause().getMessage().startsWith("msg HTTP error 404"),
-            res2.cause().getMessage());
-        }
-        dh.postUrlJson("/version", "msg", "{}", res3 -> { // provoke 404 not found
-          context.assertTrue(res2.failed());
-          if (res3.failed()) {
-            context.assertTrue(res3.cause().getMessage().startsWith("msg HTTP error 404"),
-              res3.cause().getMessage());
-          }
-          async.complete();
-        });
-      });
+      async.complete();
     });
+    async.await(1000);
+    Assume.assumeTrue(versionRes.containsKey("result"));
+    context.assertTrue(versionRes.getJsonObject("result").containsKey("Platform"));
```

Comment:
Can you remove this line or tweak it? It failed on one of our ec2 box because "Platform" does not exist. The actual result is `{"Version":"17.06.2-ce","ApiVersion":"1.30","MinAPIVersion":"1.12","GitCommit":"402dd4a/17.06.2-ce","GoVersion":"go1.8.3","Os":"linux","Arch":"amd64","KernelVersion":"4.14.94-89.73.amzn2.x86_64","BuildTime":"2017-11-14T22:04:39.163310834+00:00"}`